src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -441,6 +441,10 @@ public string TargetedRuntimeVersion
             set { _targetedRuntimeVersionRawValue = value; }
         }
 
+        public string PreComputedCacheOutputPath { get; set; }
```

Comment:
I don't like this much better, would like to think on a name for these.

```suggestion
        public string OutputCache { get; set; }
```

Reply:
How about CacheOutputPath, Relative(Path)CacheOutputPath, or OutputCachePath?